modules/wolfgame.py

```diff
@@ -787,6 +795,189 @@ def fpinger(cli, nick, chan, rest):
     var.LAST_PING = None
     pinger(cli, nick, chan, rest)
 
+@cmd("pingif", "pingme", "pingat", pm=True)
+def altpinger(cli, nick, chan, rest):
+    """Pings you when the number of players reaches your preference."""
+    if not rest or (rest.isdigit() and int(rest) == 0):
```

Comment:
Not entirely sure that having the param blank should be considered the same as clearing it. It may be more useful for an empty !pingif to simply tell you what your current preference is.